### DISCUSSION TITLE
Fivem activity pub feed Error

### DISCUSSION THREAD

**Discussion by capo2001** [mastodon/mastodon#33012] [General]
i have the problem that i cant sync my account whit Fivem 

server.cfg 

sets activitypubFeed Haupstadt@mastodon.social
![Screenshot 2024-11-21 090532](https://github.com/user-attachments/assets/70be385d-2aa0-40d3-9e6d-fa7ee439d4fd)

**Comment by Honeybee-Systems** [comment]
same
